### PR TITLE
Add documentation for onCompleteOverride and list argumetns for onCom…

### PR DESCRIPTION
**Pull request by garvank**
…plete

### CODE REVIEW

**Comment by garvank** [comment]
@nburwell Could you please review? I think all we wanted to add was `onCompleteOverride` and add the arguments it accepts. Happy to add anything else we need.

File: source/web_integration/pnapi_web_integration.rst [hunk 1]
```diff
@@ -183,10 +183,22 @@ Client Side Parameters
...
 
       For example: `functionName` not `functionName()`.
 
+  * - onCompleteOverride
+    - Function
+    - Default: `null`
+
+      Arguments: `batchId`, `mapping`
+
+      Name of a function to call when Invoca has finished running.
```
Comment (nburwell) [inline source/web_integration/pnapi_web_integration.rst]: I would mention this one:

```
Name of a function to call when Invoca has finished running. If defined, the function is responsible for updating phone numbers on the web page as the default number replacement code will not be run.
```

File: source/web_integration/pnapi_web_integration.rst [hunk 2]
```diff
@@ -183,10 +183,22 @@ Client Side Parameters
     - Function
     - Default: `null`
 
+      Arguments: `mapping`
+
       Name of a function to call when Invoca has finished running. Note: Do not call the function, simply pass reference.
```
Comment (nburwell) [inline source/web_integration/pnapi_web_integration.rst]: Given the `onCompleteOverride` function is now also defined, how about clarifying this one to something like:

```
Name of a function to call when Invoca has finished running.  This is called after Invoca has run the default number replacement code and swapped out any numbers that were found. Note: Do not ...
```

**Review by nburwell — COMMENTED** [review]
Minor comments on the help copy

**Review by nburwell — APPROVED** [review]
Looks good

**Pull request merged**